README.md

```diff
@@ -22,6 +22,11 @@ Tools:
 * Ray Tune 
 * ELI5, SHAP
 
+## Notebooks
+
+<a href="https://nbviewer.jupyter.org/github/PacktPublishing/Production-Ready-Applied-Deep-Learning/blob/main_readme/index.ipynb"><img src="https://raw.githubusercontent.com/jupyter/design/master/logos/Badges/nbviewer_badge.svg" alt="Render nbviewer" /></a>
+
+
```

Comment:
For the author section, can we use markdown styling with only one link?
I don't think gitlab account add much value. or does it?

[Tomasz (Tom) Palczewski](https://www.linkedin.com/in/tomaszpalczewski/)

I'd like to link my personal page (https://ljj7975.github.io/)